packages/wrangler/package.json

```
@@ -70,13 +70,14 @@
 	},
 	"dependencies": {
 		"@cloudflare/kv-asset-handler": "workspace:*",
+		"@cloudflare/unenv-preset": "2.0.0",
 		"@esbuild-plugins/node-globals-polyfill": "0.2.3",
 		"@esbuild-plugins/node-modules-polyfill": "0.2.2",
 		"blake3-wasm": "2.1.5",
 		"esbuild": "0.17.19",
 		"miniflare": "workspace:*",
 		"path-to-regexp": "6.3.0",
-		"unenv": "2.0.0-rc.1",
+		"unenv": "2.0.0-rc.8",
```

Perhaps move this to the pnpm workspace catalog?

```suggestion
		"unenv": "catalog:default",
```

---

Since the vite plugn will use the same version

---

> Since the vite plugn will use the same version

Can this be done when the cute plugin is ready? 
I have no idea about when it's ready to move to rc8